scripts/report-ci-preflight-results/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+func main() {
+	if err := run(); err != nil {
+		time.Sleep(50 * time.Millisecond) // let stdout flush
```

Comment:
How does it work?

Reply:
I want the error that's printed on the next line to at the very end of the output, because it will have the command to rerun things. stdout/stderr being unbuffered and the way that github actions capture those two output streams means it's not guaranteed to be at the end of the github action output. adding this little sleep was my i'm a lazy engineer and this will mostly work fix :-) 

this is what happened before I put in this short sleep: https://github.com/superfly/flyctl/actions/runs/5115901837/jobs/9197567462

Reply:
Because we have a lot in stdout? I could take a look that after master merge.

Reply:
I deleted all this after finding a somewhat simpler way to avoid logging the history output. So it should all be moot now :-)